src/constants.js

```diff
@@ -20,6 +20,7 @@ exports.LOGIN_PATH = '/login';
 exports.LOG_OUT_PATH = '/log-out';
 exports.SETTINGS_PATH = '/settings';
 exports.DEBUG_PATH = '/debug';
+exports.RULES_ENGINE_PATH = '/rules-engine';
```

Comment:
If the rules engine is meant to be a pluggable module, it seems strange to hard code this here.